### ISSUE TITLE
Request : being able to import / export settings

### ISSUE THREAD

**Issue by mijkami** [beingmrkenny/ListHighlighterForTrello#4]

**Comment by beingmrkenny** [comment]
Thanks! This is on my list :)

**Comment by jj-dominguez** [comment]
Yeah definitely this! or sync settings between two machines. I am absolutely loving your extension, as someone who is really visual and easily distracted it's insane how much this has helped me to focus on the things that matter. I just wish I could sync the settings, I get it set up perfectly on my personal comp then switch to work and have to tweak things to get it consistent.

Oh and also being able to disable it for specific boards, there are a few where I don't need/want the same styling.

If you left some notes/documentation on how you specifically would go about achieving any of these things I might be able to cut a PR for ya, let me know.

**Comment by beingmrkenny** [comment]
Hi,

Syncing should happen automatically if you are signed into your browser with the same account. Obviously, this only applies if you are using the same browser on each machine (e.g. Chrome on your home and work computers).

If you use Chrome at work and Firefox at home, there's not currently a way to sync settings automatically, but I do have plans to allow a copy/paste functionality, or to download and upload settings file with merging of rules, including the ability to resovle conflicts neatly. As you can probably appreciate this is a lot of work!

I've added the ability to distinguish between boards to my feature request list :)
